src/main/java/org/reaktivity/specification/kafka/internal/KafkaFunctions.java

```diff
@@ -1341,6 +1407,19 @@ public KafkaFetchDataExMatcherBuilder partition(
                 return this;
             }
 
+            public KafkaFetchDataExMatcherBuilder partition(
```

Comment:
Please call this from the 2-parameter overloaded `partition` method, passing `DEFAULT_LATEST_OFFSET` for the `latestOffset` argument.